### PR TITLE
#4121 Added Cibo Espresso to amenity/cafe

### PR DESCRIPTION
**Pull request by ParthoShuvo**

### CODE REVIEW

File: brands/amenity/cafe.json [hunk 1]
```diff
@@ -430,6 +430,20 @@
...
     },
+    {
+      "displayName": "Cibo Espresso",
+      "id": "ciboespresso-70e4e4",
+      "locationSet": {"include": ["au"]},
+      "tags": {
+        "amenity": "cafe",
+        "brand": "Cibo Espresso",
+        "brand:wikidata": "Q5119230",
+        "brand:wikipedia": "en:Cibo Espress",
```
Comment (bhousel) [inline brands/amenity/cafe.json:441]:
```suggestion
        "brand:wikipedia": "en:Cibo Espresso",
```

**Review by bhousel — CHANGES_REQUESTED** [review]
Welcome @ParthoShuvo  👋 
Looks mostly good - just one minor issue to fix before we can merge it..

**Comment by ParthoShuvo** [comment]
Hi @bhousel,

Please review the changes.

Thanks.

**Review by bhousel — APPROVED** [review]

**Comment by bhousel** [comment]
Looks good @ParthoShuvo , thanks for the quick fix!

**Pull request merged**